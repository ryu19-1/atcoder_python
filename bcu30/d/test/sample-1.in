5 4
1 4 5 8 10
3
4
7
11
