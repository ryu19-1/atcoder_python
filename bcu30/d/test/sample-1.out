17
14
15
27
